lib/App.js

```diff
@@ -1444,7 +1442,7 @@ class App {
         appJson = appJson.toString();
         appJson = JSON.parse(appJson);
         appJson.drivers = appJson.drivers || [];
-        appJson.drivers.push( driverJson );
+        appJson.drivers.push({ id: driverId, ...driverJson });
```

Comment:
Was the ID previously not set?

Reply:
no its the opposite, I removed the id from `driverJson` but in this case (when homeycompose is not used) we still need it so I add it back into the object

Reply:
Ah i see.